File: go/wam/1384.go
const WamUserActivity = 1384 // channel: regular

type UserActivityEvent struct {
	UserActivitySessionId string `wam:"user_activity_session_id"` // field 1
	UserActivityStartTime int64 `wam:"user_activity_start_time"` // field 2
	UserActivityBitmapLow int64 `wam:"user_activity_bitmap_low"` // field 3
	UserActivityBitmapHigh int64 `wam:"user_activity_bitmap_high"` // field 4
	UserActivityBitmapLen int64 `wam:"user_activity_bitmap_len"` // field 5
	UserActivitySessionSeq int64 `wam:"user_activity_session_seq"` // field 6
	UserActivitySessionCum int64 `wam:"user_activity_session_cum"` // field 7
}

File: ts/wam/1384.ts
export const WamUserActivity = 1384 as const

export interface UserActivityEvent {
  /** field 1, wire `user_activity_session_id` */
  userActivitySessionId?: string
  /** field 2, wire `user_activity_start_time` */
  userActivityStartTime?: number
  /** field 3, wire `user_activity_bitmap_low` */
  userActivityBitmapLow?: number
  /** field 4, wire `user_activity_bitmap_high` */
  userActivityBitmapHigh?: number
  /** field 5, wire `user_activity_bitmap_len` */
  userActivityBitmapLen?: number
  /** field 6, wire `user_activity_session_seq` */
  userActivitySessionSeq?: number
  /** field 7, wire `user_activity_session_cum` */
  userActivitySessionCum?: number
}